### DISCUSSION TITLE
Homebrew not installing on Big Sur v 11.2.1 - Failed during: git fetch --force origin

### DISCUSSION THREAD

**Discussion by c-lewis99** [Homebrew/discussions#814] [Getting started]
Already have the latest Xcode Command line tools installed. 
However when I try to install Homebrew using;
 
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"

I get the error;

fatal: the remote end hung up unexpectedly8.39 MiB | 1.04 MiB/s 
fatal: early EOF
fatal: index-pack failed
Failed during: git fetch --force origin

Have tried uninstalling and reinstalling it but keep getting the same error. 
Is this a connections/permissions/software issue?

Any help appreciated!
Cheers

**Comment by SMillerDev** [comment]
This is a connection issue with Github you could check https://github.com/Homebrew/discussions/discussions/666#discussioncomment-330855 for more info

**Reply by c-lewis99** [reply]
Have tried pretty much all the suggestions I've come across (changing git buffer size, trying SSH etc) to no avail.
Could it be my wifi? Have already tried with my 4g network.

**Reply by unnithekkekuruvath** [reply]
My issue turned out to be DNS related, I set my network config to Google's public 8.8.8.8 DNS. I can now install Homebrew via terminal with no issues.

**Reply by leonardkorb** [reply]
> My issue turned out to be DNS related, I set my network config to Google's public 8.8.8.8 DNS. I can now install Homebrew via terminal with no issues.

it was the only thing that worked for me. Thank you so much for posting.

**Reply by haddagart** [reply]
Changing my DNS to Google's 8.8.8.8 public DNS worked for me. I almost changed my .gitconfig credentials  if I wasn't sure about it. Thank you so much for this trick.
